TROUBLESHOOTING.md

```diff
@@ -74,6 +74,28 @@
     2. Delete the offending directories from your object storage location.
 
 
+### Known issue with Backup/Restore using Restic:
```

Comment:
change this to `Known issue with backup/restore of DeploymentConfigs using Restic`